website/docs/sdk-reference/unreal.mdx

```diff
@@ -24,19 +24,19 @@ export const CPPSchema = require('@site/src/schema-markup/sdk-reference/cpp.json
 
 Via **[Unreal Marketplace](https://www.unrealengine.com/marketplace/product/e142293a397d4ce4bf6a1f3053a2316d)**
 
-Via **[GitHub clone](https://github.com/configcat/unreal-engine-sdk)**
+Via **[GitHub](https://github.com/configcat/unreal-engine-sdk)**
 
-Prequesities to cloning manually:
+Prequesities to building manually:
 - you are working in a [C++ project](https://docs.unrealengine.com/5.2/en-US/compiling-game-projects-in-unreal-engine-using-cplusplus/)
 - you've completed the [Visual Studio](https://docs.unrealengine.com/5.2/en-US/setting-up-visual-studio-development-environment-for-cplusplus-projects-in-unreal-engine/) or [Visual Studio Code](https://docs.unrealengine.com/5.2/en-US/setting-up-visual-studio-code-for-unreal-engine/) setup
 
-Run the following commands in the root folder of your project (where the `.uproject` file is located):
+To download:
 
-```cmd
-mkdir -p Plugins
-cd Plugins
-git clone https://github.com/configcat/unreal-engine-sdk ConfigCat
-```
+- Create a `Plugins` folder in the root folder of your project (where the `.uproject` file is located).
+- Head to the download the latest [ConfigCat.zip](https://github.com/configcat/unreal-engine-sdk/releases/latest/download/ConfigCat.zip) archive. (or any of the previous versions from: [Releases](https://github.com/configcat/unreal-engine-sdk/releases))
```

Comment:
Just a suggestion, I think it is not necessary to put the other part in brackets.
```suggestion
- Head to the download the latest [ConfigCat.zip](https://github.com/configcat/unreal-engine-sdk/releases/latest/download/ConfigCat.zip) archive or any of the previous versions from: [Releases](https://github.com/configcat/unreal-engine-sdk/releases)
```